File: tools/weixin/src/main/java/top/bettercode/summer/tools/weixin/config/MiniprogramProperties.java
package top.bettercode.summer.tools.weixin.config;

import org.springframework.boot.context.properties.ConfigurationProperties;

/**
 * @author Peter Wu
 */
@ConfigurationProperties(prefix = "summer.wechat.mini")
public class MiniprogramProperties extends WexinProperties implements IMiniprogramProperties{


}

File: tools/weixin/src/main/kotlin/top/bettercode/summer/tools/weixin/properties/MiniprogramProperties.kt
package top.bettercode.summer.tools.weixin.properties

import org.springframework.boot.context.properties.ConfigurationProperties
import top.bettercode.summer.tools.weixin.support.miniprogram.entity.MiniprogramState

/**
 * @author Peter Wu
 */
@ConfigurationProperties(prefix = "summer.wechat.mini")
open class MiniprogramProperties : WexinProperties(), IMiniprogramProperties {
    /**
     *  跳转小程序类型：developer为开发版；trial为体验版；formal为正式版；默认为正式版
     */
    override var miniprogramState: String = MiniprogramState.formal
}
